templates/cli/base/requests/api.twig

```diff
@@ -17,8 +17,19 @@
     fs.writeFileSync(destination, response);
     {%~ endif %}
     if (parseOutput) {
+    {%~ if method.name == 'get' and service.name not in ['health','migrations','locale'] %}
```

Comment:
Can we make this supoprt list too? `appwrite functions list` or `appwrite functions listExecutions`